app/models/draftMovements/GetDraftMovementsSearchOptions.scala

```diff
@@ -107,11 +114,62 @@ object GetDraftMovementsSearchOptions extends Logging {
           Some(intBinder.unbind("index", searchOptions.index)),
           searchOptions.searchTerm.map(searchTerm => stringBinder.unbind("searchTerm", searchTerm)),
           searchOptions.draftHasErrors.map(hasErrors => booleanBinder.unbind("draftHasErrors", hasErrors)),
-          searchOptions.destinationTypes.map(destinationTypes => destinationTypeBinder.unbind("destinationType", destinationTypes)),
+          searchOptions.destinationTypes.map(destinationTypes => destinationTypeBinder.unbind("destinationTypes", destinationTypes)),
           searchOptions.dateOfDispatchFrom.map(date => stringBinder.unbind("dateOfDispatchFrom", date.toString)),
           searchOptions.dateOfDispatchTo.map(date => stringBinder.unbind("dateOfDispatchTo", date.toString)),
           searchOptions.exciseProductCode.map(code => stringBinder.unbind("exciseProductCode", code))
         ).flatten.mkString("&")
       }
     }
+
+  def apply(
+             sortBy: String,
+             searchTerm: Option[String],
+             errors: Set[DraftMovementsErrorsOption],
+             destinationType: Set[DestinationType],
```

Comment:
This one, however, probably would make sense to change to plural, as it is now represented as a `Set[]` at this point once each queryParam has been bound to the model.